packages/esm-patient-chart-app/src/ui-components/notifications-button.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from 'carbon-components-react/es/components/Button';
+import { Notification20 } from '@carbon/icons-react';
```

Comment:
Can we use direct submodule import?

Reply:
https://github.com/openmrs/openmrs-esm-patient-chart/pull/259/commits/7177949c0c470475961e8afb5c1347279d94b499